src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java

```diff
@@ -2,6 +2,7 @@
 
 import java.lang.reflect.Method;
 
+import org.apache.http.client.methods.HttpRequestBase;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.http.client.methods.HttpRequestBase'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)